scripts/magicgen-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Could you please add `set -x`? This would make it easier to debug if the test fails in CI.